CMakeLists.txt.in

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 3.5)
+project(build-seastar NONE)
+
+include(ExternalProject)
+
+ExternalProject_Add(seastar_ext
```

Comment:
oh wow so many of these are such a straight translation from cooking_ingredient.